OutSystems/outsystems-ui
# ROU-4250: Implement Video pattern with new architecture

This PR is for adding the new architecture to Video Pattern.

### Test Steps
1. Open Sample Page
2. Set the configs of Video
3. Click on Apply Button
4. Expected: The configs will be applied to Video

### Checklist
